godtools/App/Features/Menu/Presentation/Tutorial/TutorialView.swift

```diff
@@ -60,10 +60,10 @@ struct TutorialView: View {
                     attributes: pageControlAttributes,
                     currentPage: $viewModel.currentPage
                 )
+                .flipsForRightToLeftLayoutDirection(true)
```

Comment:
Hey @gyasistory this might be redundant here since PageControl.swift contains the .flipsForRightToLeftLayoutDirection modifier on it here -> https://github.com/CruGlobal/godtools-swift/blob/develop/godtools/App/Share/SwiftUI%20Views/PageControl/PageControl.swift#L63